queries/__init__.py

```diff
@@ -51,6 +51,22 @@
 LatLonTuple = Tuple[float, float]
 
 
+MONTH_ABBREV: Sequence[str] = (
```

Comment:
Can this be a frozenset() (more efficient for lookups), or should there perhaps be both a set and a list/tuple?

Reply:
Added a frozenset for lookups, kept the list also for indexing.